README.md

```diff
@@ -37,15 +42,19 @@ Examples
   port = 8443
   username = foo
   password = bar
-  set-routes = 0
+  # The pinentry program to use. Allows supplying the password in a secure manner. See the man page for details.
+  # pinentry = pinentry-mac
```

Comment:
It is not a good idea to comment this option. These examples show how it **is** done in specific cases, not how it **may** be done.

What about creating a short example of its own for `pinentry?. Call it "Using pinentry to supply the password securely" or something like that. Perhaps just after the second example 'Connect to a VPN using an authentication realm".